content-security-policy/script-src/script-src-strict_dynamic_discard_whitelist.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <title>Whitelists are discarded with `strict-dynamic` in the script-src directive.</title>
+    <script src='/resources/testharness.js' nonce='dummy'></script>
+    <script src='/resources/testharnessreport.js' nonce='dummy'></script>
+
+    <!-- CSP served: script-src 'self' 'strict-dynamic' 'nonce-dummy' -->
+</head>
+
+<body>
+    <h1>Whitelists are discarded with `strict-dynamic` in the script-src directive.</h1>
+    <div id='log'></div>
+
+    <script id='whitelistedScript' src='simpleSourcedScript.js'></script>
+    <script nonce='dummy'>
+        async_test(function(t) {
+            document.addEventListener('securitypolicyviolation', t.step_func_done(function(e) {
+                assert_equals(document.getElementById('whitelistedScript').getAttribute('executed'), null);
+                assert_equals(e.violatedDirective.split(' ')[0], 'script-src');
```

Comment:
I mean, I know I could use substitution, but I would have to do that conditionally, just for FF. I would rather just check the directive alone, as other tests do. WDYT?